### DISCUSSION TITLE
i18n links in dropdown, and footer

### DISCUSSION THREAD

**Discussion by tomy0000000** [jpanther/congo#290] [Ideas]
Having translation links next to the website just doesn't feel right for me, so I manage to move it down to the footer.

Also make it a dropdown menu instead of rows of links, which should be more friendly if the site had multiple languages.

https://user-images.githubusercontent.com/23290356/188319075-ab38a191-3695-4675-b334-d9aeb47531b7.mov

Please let me know if this is something you'll want to have in congo, and I might spent some time drafting a PR.

**Comment by jpanther** [comment]
Thanks, this is a nice feature. I'd be interested to see a PR for this if you have time. I'm assuming you're using Tailwind classes where you can?

**Comment by AntoineSoetewey** [comment]
Hello @tomy0000000,

There is currently some discussions about the position of the language switcher (see https://github.com/jpanther/congo/issues/583 and https://github.com/jpanther/congo/discussions/462).

Your solution to have the language switcher in the footer is a step in the desired direction. By any chance, would you have the time to propose a PR?

Regards,
Antoine

**Reply by tomy0000000** [reply]
Thanks for bringing this to my attention. I'll leave my feedback in #583.
